README.md

```diff
@@ -26,6 +26,35 @@ Or install it yourself as:
     PolishNumber.translate(34, :currency => :PLN) #=> trzydzieści cztery złote
     PolishNumber.translate(12, :currency => :PLN) #=> dwanaście złotych
 
+## Cents format
+
+    PolishNumber.translate(34.11) #=> trzydzieści cztery i jedenaście setnych
+    PolishNumber.translate(30.01, :currency => :PLN) #=> trzydzieści złote i jeden grosz
+    PolishNumber.translate(0.10, :currency => :PLN) #=> dziesięć groszy
+    PolishNumber.translate(5.10, :currency => :PLN, :cents => :no ) #=> pięć złotych
+    PolishNumber.translate(5, :currency => :PLN, :cents => :words ) #=> pięć złotych zero groszy
+    PolishNumber.translate(5, :currency => :PLN, :cents => :digits ) #=> pięć złotych 00/100
+
+## Adding new currencies
+
+    PolishNumber.add_currency(:COWS, { :one => 'krowa', :few => 'krowy',
```

Comment:
This isn't a currency. It's a unit. Can we keep `:currency` in `.translate` for backwards compatibility? Can we add there `:unit` as an alias to `:currency`? Can we rename this method to `.add_unit`?